factory/declarations.py

```diff
@@ -393,7 +393,7 @@ def get(self):
 
     def __repr__(self):
         if self.factory is None:
-            return '<_FactoryImport: %s.%s>' % (self.module, self.name)
+            return f'<_FactoryImport: {self.module}.{self.name}>'
         else:
             return '<_FactoryImport: %s>' % self.factory.__class__
```

Comment:
For consistency, I'd add a f-string here too

Reply:
Done in the latest revision.